debian/README.Debian

```diff
@@ -0,0 +1,17 @@
+chicago95 for Devuan
```

Comment:
if this is debian should it not say debian?

Reply:
The packages have been built and tested against Devuan. It should be fine to change the documentation to say Debian. I expect it to work better against debian. I don't change the filenames of the debian packaging tools regardless of which OS it's for.